requirements.txt

```diff
@@ -1,6 +1,7 @@
 click==6.2
 PTable==0.9.2
 netaddr==0.7.18
+nsot==1.1.1
```

Comment:
My entire worry was that the two `slugify` functions would fall out of sync, but I forgot that we run an instance of the nsot server during tests, so as long as that version's kept up to date in `requirements-dev.txt`, that should catch any breaking disparity.